auraed/src/runtime/cells/cell.rs

```diff
@@ -74,11 +108,26 @@ impl Cell {
         };
 
         // Start the child process
-        let exe = Executable::spawn(command)?;
+        let exe =
+            Executable::start(exe_name.clone(), command).map_err(|e| {
```

Comment:
why wouldn't Executable::start (part of the cells mod) return an ExecutableError directly (and avoid the map)?

Reply:
The executable doesn't know what cell it is in. On the map, the cell wraps the executable error in its own (`CellError`) and adds the name of the cell.

Lets not talk about recursion just yet, lol